benchmarks/BaselinesTcp.cpp

```diff
@@ -0,0 +1,178 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <arpa/inet.h>
+#include <folly/Benchmark.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <atomic>
+#include <cstring>
+#include <iostream>
+#include <thread>
+
+#define MAX_MESSAGE_LENGTH (8 * 1024)
+#define PORT (35437)
+
+static void BM_Baseline_TCP_SendReceive(
+    size_t loadSize,
+    size_t msgLength,
+    size_t recvLength) {
+  std::atomic<bool> accepting{false};
+  std::atomic<bool> accepted{false};
+
+  std::thread t([&]() {
+    int serverSock = socket(AF_INET, SOCK_STREAM, 0);
+    int sock = -1;
+    struct sockaddr_in addr;
+    socklen_t addrlen = sizeof(addr);
+    char message[MAX_MESSAGE_LENGTH];
+
+    std::memset(message, 0, sizeof(message));
```

Comment:
we have seen odd behavior when the buffer was uninitialized. I never got into trying to understand why so and am just keeping it as it.